pkg/api/v20170930/types.go

```diff
@@ -0,0 +1,16 @@
+package v20170930
+
+// VersionInfo contains rersion and release numbers
+type VersionInfo struct {
+	Release string `json:"release,omitempty"`
+	Version string `json:"version"`
+}
+
+// OrchestratorInfo contains orchestrator version info
+type OrchestratorInfo struct {
+	Orchestrator string `json:"orchestrator"`
+	VersionInfo
+	DockerComposeVersion string         `json:"docker-compose-version,omitempty"`
```

Comment:
Do we really need to expose this as a field in the RP? IMO no just like we don't expose heapster or etcd version.